### ISSUE TITLE
App crashes in ConceptTensorFlowObjectDetection.java when in stock mode

### ISSUE THREAD

**Issue by PotatosFish** [OpenFTC/OpenRC-Turbo#6]
I have copied ConceptTensorFlowObjectDetection from the repository and tried to run it in extreme turbo mode, and it worked perfectly. However, when I switched back to stock, the app kept crashing on init. The error that caused the crash is shown below:

```
2019-11-19 17:03:33.896 18853-19189/com.qualcomm.ftcrobotcontroller E/AndroidRuntime: FATAL EXCEPTION: LinearOpMode main
    Process: com.qualcomm.ftcrobotcontroller, PID: 18853
    java.lang.NoSuchMethodError: No direct method <init>(Ljava/nio/MappedByteBuffer;I)V in class Lorg/tensorflow/lite/Interpreter; or its super classes (declaration of 'org.tensorflow.lite.Interpreter' appears in /data/app/com.qualcomm.ftcrobotcontroller-2/base.apk)
        at org.firstinspires.ftc.robotcore.internal.tfod.TFObjectDetectorImpl.initialize(TFObjectDetectorImpl.java:267)
        at org.firstinspires.ftc.robotcore.internal.tfod.TFObjectDetectorImpl.loadModelFromFile(TFObjectDetectorImpl.java:247)
        at org.firstinspires.ftc.teamcode.ASDF.initTfod(ASDF.java:165)
        at org.firstinspires.ftc.teamcode.ASDF.runOpMode(ASDF.java:91)
        at com.qualcomm.robotcore.eventloop.opmode.LinearOpMode$LinearOpModeHelper$1.run(LinearOpMode.java:305)
        at com.qualcomm.robotcore.util.ThreadPool.logThreadLifeCycle(ThreadPool.java:737)
        at com.qualcomm.robotcore.eventloop.opmode.LinearOpMode$LinearOpModeHelper.run(LinearOpMode.java:300)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1133)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:607)
        at com.qualcomm.robotcore.util.ThreadPool$ThreadFactoryImpl$1.run(ThreadPool.java:793)
        at java.lang.Thread.run(Thread.java:761)
```

I tried rebuilding it, invalidating caches, and nothing worked.

From what I can decipher from the error, it's probably that the tensorflow lite library changed in between the build variant, but I do not know how to fix it.

My android studio version is 3.5.2, and OS is macOS Mojave.

**Comment by FROGbots-4634** [comment]
Yikes. Will look into this as soon as I can.

**Comment by FROGbots-4634** [comment]
I was able to reproduce this with OpenRC 5.3A. I will look into the cause.

**Comment by FROGbots-4634** [comment]
I have identified the cause. A new release will be pushed ASAP.

**Comment by FROGbots-4634** [comment]
Fixed in 5.3B release.
